.github/workflows/staging.yml

```diff
@@ -39,8 +39,8 @@ jobs:
         service: [ 'backend', 'employer' ]
         include:
           - service: backend
-            context: ./backend
-            dockerfile: ./backend/Dockerfile
+            context: ./backend/kesaseteli
```

Comment:
I think we should limit this workflow to only run when there are changes in kesaseteli folder. In helsinki-benefit we use something like this: https://github.com/City-of-Helsinki/yjdh/pull/46/files#diff-280994542061213299f64fa399afda4ec4a787875fe7953d3595900eb09cbc40R6